psi4/src/psi4/libmints/basisset.h

```diff
@@ -390,7 +396,7 @@ class BasisSet
      * @param py::dict      Python dictionary containing the basis information
      * @param forced_puream Force puream or not
     **/
-    static std::shared_ptr<BasisSet> construct_from_pydict(const std::shared_ptr <Molecule> &mol, py::dict pybs, const int forced_puream);
+    static std::shared_ptr<BasisSet> construct_from_pydict(const std::shared_ptr <Molecule> &mol, py::dict &pybs, const int forced_puream);
```

Comment:
Declaring fn in class header but not implementing it until export_mints — I didn't know you could do that.